app/controllers/AccountHomeController.scala

```diff
@@ -45,7 +49,25 @@ class AccountHomeController @Inject() (
     with I18nSupport
     with Retrievals {
 
-  def onPageLoad: Action[AnyContent]       = (identify andThen getData)(implicit request => Ok(view()))
+  private val logger = Logger(this.getClass)
+
+  // represents the backend retrieval
+  def onPageLoad: Action[AnyContent]       =
+    (identify andThen getData).async {
+      implicit request =>
+        val appSumReq = ApplicationSummaryRequest(EORI(request.eoriNumber))
+
+        backendConnector
+          .applicationSummaries(appSumReq)
+          .map {
+            case Right(response)    =>
+              Ok(view(response.summaries.map(ApplicationForAccountHome(_))))
+            case Left(backendError) =>
+              logger.error(s"Failed to submit user answers to backend: $backendError")
```

Comment:
I suspect the log message needs updating.

Reply:
Oops!